pkg/kubelet/client/kubelet_client.go

```diff
@@ -65,20 +65,30 @@ type KubeletClientConfig struct {
 
 // ConnectionInfo provides the information needed to connect to a kubelet
 type ConnectionInfo struct {
-	Scheme    string
-	Hostname  string
-	Port      string
-	Transport http.RoundTripper
+	Scheme                         string
+	Hostname                       string
+	Port                           string
+	Transport                      http.RoundTripper
+	InsecureSkipTLSVerifyTransport http.RoundTripper
 }
 
 // ConnectionInfoGetter provides ConnectionInfo for the kubelet running on a named node
 type ConnectionInfoGetter interface {
 	GetConnectionInfo(ctx context.Context, nodeName types.NodeName) (*ConnectionInfo, error)
 }
 
-// MakeTransport creates a RoundTripper for HTTP Transport.
-func MakeTransport(config *KubeletClientConfig) (http.RoundTripper, error) {
-	tlsConfig, err := transport.TLSConfigFor(config.transportConfig())
+// makeTransport creates a RoundTripper for HTTP Transport.
+func makeTransport(config *KubeletClientConfig, insecureSkipTLSVerify bool) (http.RoundTripper, error) {
```

Comment:
I'd prefer you didn't make this private.

Reply:
> I'd prefer you didn't make this private.

exposed as two explicit methods.  `MakeTransport` - creates a secure transport as before.  `MakeInsecureTransport` produces an insecure transport.  The diff of diffs is clean: https://github.com/kubernetes/kubernetes/compare/b855d99d58397c4a8704091bf34b7ee7e2458c54..867ee1d5ff88b42261c1224813779a66318d70b2

Reply:
thanks